polynote-frontend/polynote/ui/component/toolbar.ts

```diff
@@ -298,8 +298,8 @@ class SettingsToolbarUI extends UIMessageTarget {
                 .withKey('neverDisabled', true),
         ]]);
 
-        this.floatingMenu = div(['floating-menu'], []);
+        //this.floatingMenu = div(['floating-menu'], []);
 
-        this.el.appendChild(this.floatingMenu)
+        //this.el.appendChild(this.floatingMenu)
```

Comment:
Unnecessary comments?

Reply:
will delete... just wasn't sure it actually wasn't used for something 😛

Reply:
Guess we will find out! :P